### PR TITLE
Additional player fields, fixes #56

### LINKED ISSUE #56: Ergänzung Spieler-Profil
Zwei zusätzliche Rubriken:
"Fußballerisches Vorbild" und "Mein persönlicher CFB-Moment"

### PR DESCRIPTION
**Pull request by DenniJensen**

### CODE REVIEW

File: app/controllers/team_players_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class TeamPlayersController < ApplicationController
+  before_action :authorize_admin!
```
Comment (elgrillo83) [inline app/controllers/team_players_controller.rb]: Please favor the access-granted gem, see below.

File: app/controllers/team_players_controller.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  def show
```
Comment (elgrillo83) [inline app/controllers/team_players_controller.rb]: authorize!(: read, TeamPlayers)

File: app/controllers/team_players_controller.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    @team = Team.find(params[:team_id])
+  end
+
+  def update
```
Comment (elgrillo83) [inline app/controllers/team_players_controller.rb]: authorize!(:update, TeamPlayers)

File: app/controllers/team_players_controller.rb (hunk 1 above)
Comment (elgrillo83) [inline app/controllers/team_players_controller.rb]: Just realized that TeamPlayers is not a model. So, access-granted is still possible but not that simple. Then just keep it like that ;-)

File: app/controllers/team_players_controller.rb (hunk 3 above)
Comment (DenniJensen) [inline app/controllers/team_players_controller.rb]: why teamplayers? I do not have any model `TeamPlayer`

**Pull request merged**